libmscore/xml.cpp

```diff
@@ -625,6 +625,36 @@ QString Xml::xmlString(const QString& s)
       }
 
 //---------------------------------------------------------
+//   tagEscape
+//---------------------------------------------------------
+
+QString Xml::tagEscape(QString s, QString tag)
+      {
+      QString openTag = "<" + tag + ">";
+      QString openProxy = "!!" + tag + "!!";
+      QString closeTag = "</" + tag + ">";
+      QString closeProxy = "!!/" + tag + "!!";
+      s.replace(openTag, openProxy);
+      s.replace(closeTag, closeProxy);
+      s = Xml::xmlString(s);
+      s.replace(openProxy, openTag);
+      s.replace(closeProxy, closeTag);
+      return s;
+      }
+
+//---------------------------------------------------------
+//   unEscape
+//---------------------------------------------------------
+
+QString Xml::unEscape(QString s)
```

Comment:
What about using QTextDocumentFragment::fromHtml(s).toPlainText() ? It shoud take care of this an other entities. It's slow, but in the case we are using it, it should be ok.